addons/game/script_lib.hpp

```diff
@@ -31,5 +31,4 @@
 #define LIB_ui_fnc_add                        CBA_ui_fnc_add
 #define LIB_fnc_setVarNet                    CBA_fnc_setVarNet
 #define LIB_fnc_serverEvent                    acre_lib_fnc_serverEvent
-#define LIB_fnc_getCompartment                acre_lib_fnc_getCompartment
-#define LIB_fnc_isTurnedOut                    acre_lib_fnc_isTurnedOut
\ No newline at end of file
+#define LIB_fnc_getCompartment                acre_lib_fnc_getCompartment
```

Comment:
Git really likes new lines at end of file, best to enable that thing in whatever editor you use.